pages/[lng]/blog/[slug].tsx

```diff
@@ -1,7 +1,14 @@
 /* library package */
 import { FC, useState } from 'react';
```

Comment:
```suggestion
import { FC, useState } from 'react'
```

titik koma dihilangken